src/btree/bt_curnext.c

```diff
@@ -776,6 +776,11 @@ __wt_btcur_next(WT_CURSOR_BTREE *cbt, bool truncating)
 
     WT_STAT_CONN_DSRC_INCR(session, cursor_next);
 
+    /* Track next calls during HS wrapup */
+    if (F_ISSET(session, WT_SESSION_HS_WRAPUP)) {
+        session->reconcile_timeline.hs_wrapup_next_prev_calls++;
```

Comment:
The `reconcile_timeline` struct is intended to capture time-related stats only. It's not designed for tracking frequency-related stats. Might be worth considering a different place for frequency data.